examples/evmapp/src/main/additional-resources/runscripts/run.cmd

```diff
@@ -0,0 +1 @@
+java -jar com.horizen.examples.EvmApp sc_settings.conf
```

Comment:
we may remove `additional-resources` folder.
I don't see big usage of it. Check the `pom.xml`. Scripts usage there seems to be a relic and can be removed. Not sure 100%
The same for SimpleApp.